src/components/signup/SignupState.js

```diff
@@ -80,6 +79,10 @@ const Signup = ({ navigation, screenProps }: { navigation: any, screenProps: any
           // top wallet of new user
           // wait for the topping to complete to be able to withdraw
           await API.verifyTopWallet()
+          const { email: to, fullName: name } = state
+          const mnemonic = localStorage.getItem('GD_USER_MNEMONIC')
+          logger.info({ to, name, mnemonic })
```

Comment:
```suggestion
          log.info({ to, name, mnemonic })
```

`logger` is the main instance of `pino-logger`. A practice that may be helpful is to use `log` (after setting: `const log = logger.child({ from: 'SignupState' })`) so we can have a better organized logs in console.

Reply:
Nice suggestion! Done